python/basix/finite_element.py

```diff
@@ -582,7 +587,7 @@ def create_element(
     dpc_variant: DPCVariant = DPCVariant.unset,
     discontinuous: bool = False,
     dof_ordering: typing.Optional[list[int]] = None,
-    dtype: npt.DTypeLike = np.float64,
+    dtype: typing.Optional[npt.DTypeLike] = None,
```

Comment:
```suggestion
    dtype: npt.DTypeLike =  np.float64,
```

Reply:
Doing it the way we currently do it allows us to do this in DOLFINx to make Basix use the same default scalar type, which is convenient:

https://github.com/FEniCS/dolfinx/blob/5332dc9f4b97054a63264d8fa4711ed5e5ff0b1a/python/dolfinx/__init__.py#L39-L41